client/proto/daemon.proto

```
@@ -362,12 +374,16 @@ message GetConfigResponse {
 
   bool disable_ipv6 = 27;
 
+  bool serverVNCAllowed = 28;
```

Buf (1.72.0)

[error] Field "28" with name "serverVNCAllowed" on message "GetConfigResponse" changed type from "string" to "bool". See https://developers.google.com/protocol-buffers/docs/proto3#updating for wire compatibility rules and https://developers.google.com/protocol-buffers/docs/proto3#json for JSON compatibility rules.

(FIELD_WIRE_JSON_COMPATIBLE_TYPE)